LDAR_Sim/src/file_processing/input_processing/infrastructure_processing.py

```diff
@@ -38,12 +38,12 @@ def read_in_infrastructure_files(virtual_world, in_dir: Path) -> dict[str, DataF
     input_dict["sites"] = file_reader(in_dir / virtual_world["infrastructure"]["sites_file"])
 
     if virtual_world["infrastructure"]["site_type_file"]:
-        input_dict["site_types"] = file_reader(
+        input_dict[ic.Sites_File_Constants.TYPE] = file_reader(
```

Comment:
Here this refers to the presence of the file itself - Should probably be a different constant than the one for the columns header in the sites file